### ISSUE TITLE
#161: added serialOf to CLI

### ISSUE THREAD

**Issue by uweschaefer** [factcast/factcast#162]
_From @uweschaefer on 2018-10-21 12:29:20 +0000 UTC_

_Copied from original issue: Mercateo/factcast#162_

**Comment by uweschaefer** [comment]
_From @codecov[bot] on 2018-10-21 12:30:41 +0000 UTC_

# [Codecov](https://codecov.io/gh/Mercateo/factcast/pull/162?src=pr&el=h1) Report
> Merging [#162](https://codecov.io/gh/Mercateo/factcast/pull/162?src=pr&el=desc) into [master](https://codecov.io/gh/Mercateo/factcast/commit/c7e619e062f0542af2610574f31f11a425d7493c?src=pr&el=desc) will **decrease** coverage by `0.15%`.
> The diff coverage is `n/a`.

[![Impacted file tree graph](https://codecov.io/gh/Mercateo/factcast/pull/162/graphs/tree.svg?width=650&token=Kt03N0Cztt&height=150&src=pr)](https://codecov.io/gh/Mercateo/factcast/pull/162?src=pr&el=tree)

```diff
@@            Coverage Diff             @@
##             master   #162      +/-   ##
==========================================
- Coverage     78.15%    78%   -0.16%     
+ Complexity      410    409       -1     
==========================================
  Files            52     52              
  Lines          1332   1332              
  Branches        120    120              
==========================================
- Hits           1041   1039       -2     
  Misses          212    212              
- Partials         79     81       +2
```


| [Impacted Files](https://codecov.io/gh/Mercateo/factcast/pull/162?src=pr&el=tree) | Coverage Δ | Complexity Δ | |
|---|---|---|---|
| [...rg/factcast/store/pgsql/internal/PGFactStream.java](https://codecov.io/gh/Mercateo/factcast/pull/162/diff?src=pr&el=tree#diff-ZmFjdGNhc3Qtc3RvcmUtcGdzcWwvc3JjL21haW4vamF2YS9vcmcvZmFjdGNhc3Qvc3RvcmUvcGdzcWwvaW50ZXJuYWwvUEdGYWN0U3RyZWFtLmphdmE=) | `73.68% <0%> (-2.64%)` | `12% <0%> (-1%)` | |

------

[Continue to review full report at Codecov](https://codecov.io/gh/Mercateo/factcast/pull/162?src=pr&el=continue).
> **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta)
> `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> Powered by [Codecov](https://codecov.io/gh/Mercateo/factcast/pull/162?src=pr&el=footer). Last update [c7e619e...0d152ba](https://codecov.io/gh/Mercateo/factcast/pull/162?src=pr&el=lastupdated). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments).

**Comment by uweschaefer** [comment]
_From @uweschaefer on 2018-10-21 12:33:03 +0000 UTC_

closes #161
